examples/dps310/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"fmt"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/dps310"
+)
+
+const (
+	DefaultInterval = 1   // second
+	Height 			= 329 // meters, adjust for your place
+	NumMeas 		= 50
+)
+
+func blink(led machine.Pin, t, c int) {
+	for i := 0; i < c; i++ {
+		led.High()
+		time.Sleep(time.Millisecond * time.Duration(t))
+		led.Low()
+		time.Sleep(time.Millisecond * time.Duration(t))
+	}
+}
+
+func main() {
+	var (
+		err error
+	)
+
+	// Do some communiction by blinking the led
+	led := machine.LED
+	led.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	blink(led, 750, 3)
+
+	endFunc := func() {
+		blink(led, 750, 5)
+		fmt.Println("left program")
+	}
+
+	interval := time.Duration(DefaultInterval) * time.Second
+	fmt.Println("dps310 measuring interval:", interval)
+	defer endFunc()
+
+	s, err := dps310.NewSPI(
+		dps310.Config{
+			SPI: machine.SPI0,
+			CSD: machine.GPIO17,
+			SPI_Config: machine.SPIConfig{
```

Comment:
The `dps310` should probably somehow expose to the user, that only `Mode3` and a max frequency of 10 MHz is supported, or atleast check wether this holds for the config it is given.